cmd/crc/cmd/setup.go

```diff
@@ -68,7 +68,7 @@ func (s *setupResult) prettyPrintTo(writer io.Writer) error {
 
 func extraArguments() string {
 	var bundle string
-	if !constants.BundleEmbedded() {
+	if !pkgversion.IsRelease() {
```

Comment:
I am not really convince by this flag. Ideally, we should always build release binaries. I am worry that we will diverge even more over time and the CI will never test release binaries. 

Would it be possible to remove those ifs instead ? Should we invert and say `IsDev()` ?

Reply:
this `extraArgument()` change could be replaced by a check that `GetDefaultBundle()` is a valid path, but I think I would just remove the `-b ...` mention, and just indicate to use `crc start`. Then `crc start` can hint that `-b ...` should be used when it fails to find the default bundle.

Reply:
Yes no really good alternative. Let's continue with this.

Reply:
Thinking more about `IsRelease`, it's probably equivalent to `version.IsMacosInstallPathSet() || constants.BundleEmbedded()`